kebechet/managers/version/version.py

```diff
@@ -226,7 +228,13 @@ def _is_release_request(issue_title):
 
     @staticmethod
     def _compute_changelog(
-        repo: Repo, old_version: str, new_version: str, version_file: bool = False
+        repo: Repo,
+        old_version: str,
+        new_version: str,
+        changelog_smart: bool,
+        changelog_classifier: str,
+        changelog_format: str,
```

Comment:
Are these two optional?

```python
        changelog_classifier: Optional[str],
        changelog_format: Optional[str],
```

If so a check would be appropriate.

Reply:
No they aren't optional.
The arguments will always be passed on from the run() function. And run() makes sure that default classifier and formatter are selected if caller fails to specify them.

Reply:
> The arguments will always be passed on from the run() function. And run() makes sure that default classifier and formatter are selected if caller fails to specify them.

Where is this happening?

Reply:
https://github.com/thoth-station/kebechet/blob/8a711e03e95d3026174eacf40585d63d028dc447/kebechet/managers/version/version.py#L339-L348

Reply:
The input is passed from `.thoth.yaml` file sitting inside the repo. There is no check done on the actual keys present. One can easily produce wrong entries (e.g. `changelog_foo`) which will cause issues on kebechet side (and no response on the user side). I think we can introduce a mechanism to tackle this.